nanome/_internal/plugin_instance_async.py

```diff
@@ -32,8 +31,6 @@ async def async_update_loop(plugin_instance, UPDATE_RATE, MINIMUM_SLEEP):
         return
     except Exception:
         from nanome.util.asyncio import handle_exception
-        msg = traceback.format_exc()
-        logger.error(msg)
```

Comment:
This was removed because we are already logging the traceback in `handle_exception`